docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+* **Alerts related by process ancestry**: Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click **Investigate in timeline**.
+
+[discrete]
+[[prevalence-section]]
+=== Prevalence
+
+The Prevalence sub-section shows how often data from the alert was observed on other host and user events. Prevalence calculations use data from the alert's highlighted fields and examine user and host events that occurred during the last 30 days. Click **Prevalence** to display the <<prevalence-sub-tab,Prevalence sub-tab>> in the left panel. 
+
+[discrete]
+[[prevalence-sub-tab]]
+==== Prevalence sub-tab
+
+The **Prevalence** sub-tab shows relationships with other alerts, events, users, and hosts. Update the date time picker to show data from a different time range. 
 
 [role="screenshot"]
-image::images/threat-intel-sub-sec.png[Enriched with Threat Intelligence subsection with matched file hash, 700]
+image::images/prevalence-sub-tab.png[Table within the Prevalence sub-tab, 70%]
 
-TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+The following details are provided:
 
-Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
+* **Field**: Shows the alert's highlighted fields.
+* **Value**: Shows values from the alert's highlighted fields. 
+* **Alert count**: Shows the total number of alert documents that have identical field-value pairs, including the alert you're currently examining. 
++
+NOTE: Queries for the **Alert count** column only retrieve alert documents that have the `event.kind:signal` field-value pair.  
+* **Document count**: Shows the total number of event documents that contain identical field-value pairs. A dash (`——`) displays if there are no event documents that contain the field-value pair.
++
+NOTE: Queries for the **Document count** column only retrieve alert documents that do not have the `event.kind:signal` field-value pair.  
 
-IMPORTANT: After upgrading to {stack} version 7.15.0 or newer from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to instructions for <<post-upgrade-req-cti-alerts, migrating detection alerts enriched with threat intelligence data>>.
+* **Host prevalence**: Shows the percentage of unique hosts that have identical field-value pairs.  
```

Comment:
in the few bullet points above we say `the field-value pair` and here in this point and the next we say `identical field-value pairs`. Is there a reason we have single and plural?

Reply:
Rewrote this section in [f9cbf44](https://github.com/elastic/security-docs/pull/3816/commits/f9cbf44419d47058906aee164a3b490e76d9e75e). Let me know if the revised content still needs changes - thanks!